pages/en/guide/topology.md

```diff
@@ -0,0 +1,19 @@
+---
+order: 5
+---
+
+# Topology
+
+Topology is a visual representation of your instances: launched, pending, and terminated.
+Pending instances will be shown in yellow. Live instances are green, and terminated instances
+will be red. 
+
+![Topology](./imgs/topology/topology.png)
```

Comment:
Please take new screenshots based on the size and cropping of existing screenshots